blockchainetl/jobs/exporters/rabbitmq_exporter.py

```diff
@@ -0,0 +1,63 @@
+import json
+import logging
+import pika
+import pika.exceptions
+
+
+class RabbitMQItemExporter:
+    OUTPUT_FORMAT_ERROR = 'Invalid rabbitmq output param. ' \
+                   'It should be in format of "rabbitmq/{username}:{password}@localhost:5672"'
+    CONNECTION_ERROR = "Invalid output values or broker not running."
+
+    def __init__(self, output):
+        self.connection_url = self._get_connection_url(output)
+        self._connection = None
+        self._channel = None
+        self.connect()
+
+    def open(self):
+        pass
+
+    def connect(self):
+        if not self._connection or self._connection.is_closed:
+            try:
+                self._connection = pika.BlockingConnection(pika.URLParameters(self.connection_url))
+            except pika.exceptions.AMQPConnectionError:
+                raise Exception(self.CONNECTION_ERROR)
+            self._channel = self._connection.channel()
+            self._channel.queue_declare(queue='etl-queue', durable=True)
```

Comment:
I would allow configuring the queue instead of hardcoding it, maybe as a path from the connection URI?

```
rabbitmq/{username}:{password}@localhost:5672/queue-name
```